types/jss/index.d.ts

```diff
@@ -5,6 +5,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
+export interface toCSSOptions {
```

Comment:
- is it typical for typescript to start with lower case? if yes then its ok, flow convention for interfaces is capital case
- in JSS I use a convention of a consistent camel case, no matter abbreviation or not, so in case of JSS + flow we call this option `ToCssOptions`

Reply:
No, capitalization is standard in TS too.  This should be fixed.